src/config.test.ts

```diff
@@ -16,6 +16,33 @@ afterAll(() => {
   disableVerboseLogging();
 });
 
+describe("Test updateVariableWithLocalEnv function", () => {
+  beforeAll(() => {
+    process.env.hello = "world";
+    process.env.hello1 = "world1";
+  });
+  afterAll(() => {
+    delete process.env.hello;
+    delete process.env.hello1;
+  });
+  it("positive test", () => {
+    expect(
+      updateVariableWithLocalEnv("${env:hello} - ${env:hello} : ${env:hello1}")
+    ).toBe("world - world : world1");
+  });
+  it("negative test", () => {
+    try {
+      updateVariableWithLocalEnv(
+        "${env:hello2} - ${env:hello} : ${env:hello1}"
+      );
+    } catch (e) {
+      expect(e.message).toBe(
+        "Environment variable needs to be defined for hello2 since it's referenced in the config file."
+      );
+    }
+  });
+});
+
```

Comment:
there's a similar unit test located in `init.test.ts` as well :)

Reply:
I am hoping that the unit test is in config.test.ts (since the code is in config.ts) and we can do more test variations. :-)